src/errors/kinds.rs

```diff
@@ -172,6 +172,12 @@ pub enum ErrorKind {
     FrozenSetTooShort,
     #[strum(message = "FrozenSet must have at most {max_length} items")]
     FrozenSetTooLong,
+    // ---------------------
+    // timedelta errors
+    #[strum(message = "Value must be a valid timedelta")]
+    TimedeltaType,
```

Comment:
maybe this should be `TimeDeltaType`? "time delta" would never normally be two works.

and below